src/main/java/se/sundsvall/contract/integration/db/AttachmentRepository.java

```diff
@@ -0,0 +1,19 @@
+package se.sundsvall.contract.integration.db;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import se.sundsvall.contract.integration.db.model.AttachmentEntity;
+
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
+
+@CircuitBreaker(name = "attachmentRepository")
+public interface AttachmentRepository extends JpaRepository<AttachmentEntity, Long> {
+
+    List<AttachmentEntity> findAllByContractId(String contractId);
```

Comment:
What happens if the same contractId exists but for different municipalities? This method will then return contracts beloning to different municipalities, is that desired?

Reply:
See above comment =)